misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/natives/mock/util/FunctionMockUtil.java

```diff
@@ -0,0 +1,43 @@
+package org.ballerinalang.testerina.natives.mock.util;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Paths;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.ballerinalang.test.runtime.Main.getTestExecutionDependencies;
```

Comment:
Shall we move this method to `Main` class itself? since we are anyway referring to the Main class I guess this can be kept there.

Reply:
Made the `classLoader` static. So removed the implementation to resolve dependencies for function mocking. 
Removed the `FunctionMockUtil` class entirely.